### ISSUE TITLE
Weird navigationBar behavior when containing a UIPageViewController (UIViewControllerRepresentable) ?

### ISSUE THREAD

**Issue by nashfive** [faberNovel/DynamicOverlay#17]
Hi @gaetanzanella!

I have a dynamic overlay that presents a UIPageViewController via the UIViewControllerRepresentable protocol. The whole thing is presented over another view, which is contained in a fullscreenCover and a NavigationView. 
I've hidden the navigation bar on the content of my inner NavigationView (so I can push views from the dynamic overlay, inside the fullscreen's navigation context).

My problem:
The navigation bar is hidden as expected, but as soon as I swipe the UIPageViewController, the navigation bar appears. 
I can't pinpoint if the problem comes from OverlayContainer or UIPageViewController 🙁 
It only happens when I swipe the PageViewController's pages, not when I drag the overlay (it works as expected).
What do you think?  Have you come across something like this with Dynamic Overlay or OverlayContainer?

To recap, my hierarchy looks like (in pseudo code) : 
```
contentView
  .fullScreenCover(
      NavigationView {
          mapView
              .navigationBarHidden(true)
              .dynamicOverlay(OverlayView)
      }
  )

```
(OverlayView contains a swiftUI UIViewControllerRepresentable component that manages a classic UIPageViewController. Using a TabView with PageTabViewStyle was too sluggish and unoptimised for my use case)

**Comment by gaetanzanella** [comment]
could you provide a sample code? it would be great to debug the issue

**Comment by nashfive** [comment]
Hi @gaetanzanella 
Sorry for the delay... I am on vacation right now so I'll provide an example next week. But as I am not sure about the exact role of DynamicOverlay in this behavior (could be a more general issue with SwiftUI and UIKit components), I would consider this as a very low priority ;)

**Comment by nashfive** [comment]
Hi, sorry about the delay.. 
After investigation, it was not related to DynamicOverlay, nor OverlayContainer ;)
